testing/endtoend/static-files/certs/BUILD.bazel

```
@@ -0,0 +1,11 @@
+filegroup(
+    name = "web3signer-certs",
+    srcs = [
+        "cacerts.pem",
+        "knownClients.txt",
+        "pass.txt",
+        "prysm_client_identity.p12",
+        "web3signer_keystore.p12",
+    ],
+    visibility = ["//visibility:public"],
```

To prevent any of these files from being build in a production distro somehow...

```suggestion
    visibility = ["//visibility:public"],
    testonly = True,
```